.github/actions/workflow-build/action.yml

```diff
@@ -35,6 +44,20 @@ runs:
   using: "composite"
   steps:
 
+    - name: Send Slack log notification
+      if: ${{inputs.slack_token != '' && inputs.slack_log != '' }}
+      uses: slackapi/slack-github-action@v1.26.0
+      env:
+        SLACK_BOT_TOKEN: ${{ inputs.slack_token }}
+        WORKFLOW_TYPE: ${{ github.workflow }} # nightly, weekly, pr, etc.
+        SUMMARY_URL: https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}
+      with:
+        channel-id: ${{ inputs.slack_log }}
+        slack-message: |
+          Workflow '${{ env.WORKFLOW_TYPE }}' starting...
+
+          Details: ${{ env.SUMMARY_URL }}
+
```

Comment:
Question: This sends a notification every time a workflow is starting. Isn't this a lot of noise? And who is interested in these kinds of notifications? I also know a workflow is starting when I create a PR or when I add a comment `/ok to test`. What problem is this trying to solve?

Reply:
The notifications are just for nightlies/weeklies/etc, not per PR. Right now the nightly failures aren't reported anywhere.

There will be two slack channels -- one that only reports failures, and one that reports everything. The first is meant to be monitored by the team so we know when things break, the other is more of a log for looking at historical results, runtimes, startup issues, etc.

Reply:
I didn't see it was intended for the nightlies. I also perceive having a history of testing runtimes very useful. That sounds a lot more convincing! Thx!